ui/src/containers/Resources/index.tsx

```diff
@@ -17,17 +18,20 @@ import './Resources.css';
 const Resources = () => {
   const { resources } = useMst();
 
-  const checkResources = (resources: IResource[]) => {
-    return !resources.length ? (
+  const checkResources = (resourceList: IResource[]) => {
+    return !resourceList.length ? (
       <EmptyState variant={EmptyStateVariant.full} className="hub-resource-emptystate__margin">
         <EmptyStateIcon icon={CubesIcon} />
         <Title headingLevel="h5" size="md">
           No Resource Found.
         </Title>
+        <Button variant="primary" onClick={() => resources.clear()}>
+          Clear All Filters
```

Comment:
What would it show on UI if there are no resources added in the store or let's say we are getting zero resources from api, will it still show `Clear All Filters`

Reply:
nope

Reply:
Then what it would show on UI ??

Reply:
Just a spinner right ??

Reply:
yeah, for that condition we will need to handle it by `resource` api response error